### PR TITLE
fix readme badge and audit log text

### PR DESCRIPTION
**Pull request by hsiang9431-amzn**
Follow up with https://github.com/opensearch-project/security-dashboards-plugin/pull/751

### CODE REVIEW

File: public/apps/configuration/panels/audit-logging/audit-logging.tsx [hunk 1]
```diff
@@ -61,10 +61,10 @@ function renderStatusPanel(onSwitchChange: () => void, auditLoggingEnabled: bool
             <EuiText color="subdued" grow={false}>
               <FormattedMessage
                 id="audit.logs.storageInstruction"
-                defaultMessage="Configure the output location and storage types in {elasticsearchCode}. The default storage location is {internalElasticsearchCode}, which stores the logs in an index on this cluster."
+                defaultMessage="Configure the output location and storage types in {opensearchCode}. The default storage location is {internalopensearchCode}, which stores the logs in an index on this cluster."
                 values={{
-                  elasticsearchCode: <EuiCode>elasticsearch.yml</EuiCode>,
-                  internalElasticsearchCode: <EuiCode>internal_elasticsearch</EuiCode>,
+                  opensearchCode: <EuiCode>opensearch.yml</EuiCode>,
+                  internalopensearchCode: <EuiCode>internal_opensearch</EuiCode>,
```
Comment (saratvemulapalli) [inline public/apps/configuration/panels/audit-logging/audit-logging.tsx:67]: Minor nit: https://github.com/opensearch-project/opensearch-plugins/blob/main/UPGRADING.md#naming-convention
Please update L64.
```suggestion
                  internalOpenSearchCode: <EuiCode>internal_opensearch</EuiCode>,
```

**Review by saratvemulapalli — COMMENTED** [review]
Minor comments, overall the changes look good to me.

File: public/apps/configuration/panels/audit-logging/audit-logging.tsx (hunk 1 above)
Comment (hsiang9431-amzn) [reply public/apps/configuration/panels/audit-logging/audit-logging.tsx]: done

**Review by saratvemulapalli — APPROVED** [review]
LGTM!

**Pull request merged**